src/Entity/User.php

```diff
@@ -18,9 +18,27 @@ class User extends BaseUser
      */
     protected $id;
 
+    /**
+     * @var bool
+     *
+     * Flags this user as admin (adds ROLE_SUPER_ADMIN)
+     */
+    public $administrator;
```

Comment:
not sure putting this as public is a good idea, I think easy admin will skip the getters/setters

Reply:
right you are; forgot to change this while testing things out.